go/service/usersearch.go

```diff
@@ -344,6 +344,7 @@ func imptofuSearch(mctx libkb.MetaContext, provider contacts.ContactsProvider, i
 					// Ignore full-name here, force queryString as `prettyName`
 					// part in order for it to be displayed in the list.
 					imptofu.PrettyName = queryString
+					imptofu.Fullname = uname.Fullname
```

Comment:
For imptofu queries, the pretty name is set to the query string. This is done so that on the main contacts search page the subtext is the query (eg a phone number or email). But for the phone search page we want to display their full name as the subtext.